### PR TITLE
Add tax support for DKB on dividends

### PR DESCRIPTION
**Pull request by jhonnen**
This changes adds tax support for dividends only (I don't have other documents with taxes available). -> #24 

Currently the import treats the payout (i.e. net amount after taxes) as the reported amount (and calculates the per share price from that).
That seems wrong to me, the amount should be gross, not net - same as for sells.

### CODE REVIEW

File: src/brokers/dkb.js [hunk 1]
```diff
@@ -53,6 +54,22 @@ const findPayout = textArr =>
...
+  if (assessmentBasis === 0)
+    return 0;
+
+  const kap = parseGermanNum(
+    getValueByPreviousElement(textArr, 'Kapitalertragsteuer 25 %').split(' ')[0]
+  );
+  const soli = parseGermanNum(
+    getValueByPreviousElement(textArr, 'Solidaritätszuschlag').split(' ')[0]
+  );
+  return +Big(kap).plus(Big(soli));
```
Comment (dredav) [inline src/brokers/dkb.js:70]: Can you add also the church tax with the label `Kirchensteuer`?

File: src/brokers/dkb.js [hunk 2]
```diff
@@ -95,6 +113,7 @@ export const parseData = textArr => {
     amount = findAmount(textArr);
     price = findPrice(textArr);
     fee = findFee(textArr);
+    tax = 0; // TODO
```
Comment (dredav) [inline src/brokers/dkb.js:116]: I think you can also use here the `findTax` function. At some other brokers the tax structure is the same for sell and dividend documents.

File: src/brokers/dkb.js [hunk 3]
```diff
@@ -86,6 +103,7 @@ export const parseData = textArr => {
     amount = findAmount(textArr);
     price = findPrice(textArr);
     fee = findFee(textArr);
+    tax = 0.0;
```
Comment (dredav) [inline src/brokers/dkb.js:106]: Can be `0`

**Review by dredav — CHANGES_REQUESTED** [review]

File: src/brokers/dkb.js (hunk 1 above)
Comment (jhonnen) [reply src/brokers/dkb.js]: Done, also added Quellensteuer

**Comment by codeofsumit** [comment]
Great fix! Can you resolve the merge conflicts? Would be happy to merge this 👌

**Review by dredav — APPROVED** [review]

**Comment by codeofsumit** [comment]
👏 👏

**Comment by jhonnen** [comment]
Is this change live already?

**Comment by SirGibihm** [comment]
I just checked it against the version currently on tresor.one, it does not seem to be online as of yet.

**Comment by dredav** [comment]
@jhonnen @SirGibihm This change is merged, but currenly not live on production. I think this will come in the next days when Sumit is ready.

**Comment by codeofsumit** [comment]
It's live now

**Pull request merged**